README.rst

```diff
@@ -33,6 +33,7 @@ You will be prompt with questions to set configuration values.
   :db_driver_name: "postgresql", the database driver name, AnyBlok use SqlAlchemy
   :db_name: "project_name", the database name
   :http_server: ["no", "anyblok_pyramid", "anyblok_pyramid+beaker", "anyblok_pyramid+gunicorn", "anyblok_pyramid+beaker+gunicorn"], anyblok_pyramid will add a pyramid http server
+  :has_anyblok_marshmallow: ["no", "yes"], add the (de)serializator anyblok_marshmallow
```

Comment:
I would call this option **anyblok_marshmallow** (without **has_**), what do you think?